ios/FluentUI/Core/FluentUIFramework.swift

```diff
@@ -102,7 +102,7 @@ public class FluentUIFramework: NSObject {
             navigationBar.standardAppearance.backgroundColor = UIColor(dynamicColor: aliasTokens.colors[.background3])
         }
 
-        navigationBar.tintColor = UIColor(dynamicColor: aliasTokens.colors[.foreground3])
+        navigationBar.tintColor = UIColor(dynamicColor: aliasTokens.colors[.foreground2])
```

Comment:
If this is the case, then there may be many instances of navbar needing to be updated to foreground2 in devmain.

Reply:
yes it should be. it seems like figma always defined that case. title is foreground 1 and leftand right item should be foreground2
![image](https://user-images.githubusercontent.com/20715435/218880531-181f9835-ab61-4365-bc76-b0fce42281fc.png)